api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/MetricsServiceLifecycle.java

```diff
@@ -439,6 +439,7 @@ private Session createSession() {
                 .setMaxConnectionsPerHost(HostDistance.REMOTE, newMaxConnections)
                 .setMaxRequestsPerConnection(HostDistance.LOCAL, newMaxRequests)
                 .setMaxRequestsPerConnection(HostDistance.REMOTE, newMaxRequests)
+                .setMaxQueueSize(1024)
```

Comment:
@jsanda It may be useful to make this configurable.

@lucasponce As an aside, there are a variety of options used here on the clusterBuilder that perhaps we should consider?  Or, perhaps commons would be a place for some common driver settings, that could then be overridden in a component?